common/src/main/scala/explore/model/ExploreClipboard.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.effect.IO
+import explore.model.LocalClipboard.*
+import org.scalajs.dom
+
+object ExploreClipboard:
+  private val storageKey = "clipboard"
+
+  def get: IO[LocalClipboard] = IO {
+    // getItem returns null when storage isn't set.
+    Option(dom.window.localStorage.getItem(storageKey)).fold(LocalClipboard.Empty)(value =>
```

Comment:
Do we want to also verify if `localStorage` is present. in principle some browser don't have it or the user has disallowed it

Reply:
Hmmm. I can check to see how we can determine that. Then we'd have to figure out what to do if it isn't present or is disabled.

Reply:
I think it is enough to check if `localStorage` is null

Reply:
I guess it will throw an exception if it is disabled. I can try disabling it and see. Maybe we just log a message to console and in the the help for the shortcut keys say copy/paste requires that localStorage be enabled?

Reply:
Or log a message to the logger, I guess.

Reply:
OK. It seems like all of the browsers require turning off both cookies and local storage at the same time. Which would break explore, anyway. But, I suppose I should still deal with the case where local storage could be full.